src/apiCalls.js

```diff
@@ -4,10 +4,16 @@ const apiClient = axios.create({
   baseURL: 'https://nc-news-jokh.onrender.com/api',
 });
 
-export const getArticles = (topic) => {
-  let queryStr = '/articles';
+export const getArticles = (topic, sortByQuery, orderQuery) => {
+  let queryStr = '/articles?';
   if (topic) {
-    queryStr += `?topic=${topic}`;
+    queryStr += `topic=${topic}&`;
+  }
+  if (sortByQuery) {
+    queryStr += `sort_by=${sortByQuery}&`;
+  }
+  if (orderQuery) {
+    queryStr += `order=${orderQuery}`;
```

Comment:
You could refactor this to take a params object, that would negate the need for the three if conditions.